restmanager/frontend/views.py

```diff
@@ -1,44 +1,142 @@
-from django.shortcuts import render
 import os
+import json
 import slack
+import requests
 from dotenv import load_dotenv
-from django.http import HttpResponse
-from . import strings
+from fridges.models import Fridge
+from django.shortcuts import render
 from django.views.decorators.csrf import csrf_exempt
+from django.http import HttpResponse, HttpResponseRedirect
 
-# Create your views here.
+# Gets slack token from .env file and sets it here
+load_dotenv()
+slack_client = slack.WebClient(token=os.getenv("SLACK_TOKEN"))
 
-# Defining what html file to return when this function is called.
+# IP2 is the servers IP Address set in .env found in fridge_manager/restmanager
+IP2 = os.getenv('IP2', "127.0.0.1")
+# D_PORT is set only here, determines which port the get_request is sent to (the port that the server is running on)
 
+D_PORT = os.getenv('D_PORT', "8100")
 
-def fridge2(request):
-    return render(request, 'frontend/fridge2.html')
 
+@csrf_exempt
+def create_floor_list():
+    """ Creates a list of unique floors found in fridges table in the database and returns it """
+    floor_list = []
 
-def manage(request):
-    return render(request, 'frontend/index.html')
+    # adds unique floor numbers found in the get_request()
+    for item in get_request():
+        if item['floor'] not in floor_list:
+            floor_list.append(item['floor'])
+    # sorts the floor_list from smallest to largest so the list makes sense in the view
+    floor_list.sort()
```

Comment:
tän vois tehdä returnissa eikä tarttisi omaa riviä. lisäksi turha kommentti.

Reply:
Jos returnataan floor_list.sort() ei floorselect näytä yhtään flooria, näin tehtynä näyttää kerrokset ja sorttaa ne oikein.

Reply:
ah juu totta kai .sort() kun ei luo uutta eli oikea on:
return sorted(floor_list)

Reply:
Vaihdoin tohon sorted(floor_listiin) toimi ainakin lokaalisti